tests/migration_test.go

```diff
@@ -3371,12 +3378,8 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 					// Mark the masters as schedulable so we can migrate there
 					setMastersUnschedulable(false)
 
-					// Taint Node.
-					By("Tainting node with node drain key")
```

Comment:
maybe keep the By?
or even put it in the function itself (same for the other places, just the first By out of the two if there are two)

Reply:
I see no value in the `By` clauses. The name of the func - `temporarilyDrainNode` - should be enough to know it happens for a time, and by checking the details (moving to the implementation of the func) you'll know what resets the node. 

What is your motive for wanting to preserve those clauses ?

Reply:
because it is pretty intrusive act, and if the test fails we will see that it happened easier on the failing tests (if those tests that doing that are the failing ones)

Reply:
You can't trust a `By` clause to see what happened imo . 

Those are simply decorated logs sent to the `GinkgoWritter`. If you insist, I'll keep them, but I'll have to adjust what they say. Or do you want to preserve both `By` clauses ?

Reply:
I don't insist, thanks